DatabaseHandler/DatabaseController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DatabaseHandler
+{
+    public class DatabaseController
```

Comment:
Why not a controller for each table? this file is just going to keep growing in that case